### PR TITLE
Add : EMoney network Mainnet

### PR DESCRIPTION
**Pull request by deepaks-scallopx**
Add EMoney network Mainnet to chains

### CODE REVIEW

**Review by dav3yblaz3 — COMMENTED** [review]
the greatest team ever

**Review by ligi — CHANGES_REQUESTED** [review]
resolve conflicts

**Review by dav3yblaz3 — COMMENTED** [review]
lets try this

**Review by dav3yblaz3 — COMMENTED** [review]
lets do this team

File: _data/chains/eip155-4544.json
[the remark this answers is not in the document]
Comment (dav3yblaz3) [reply _data/chains/eip155-4544.json:1]: i have submitted changes lets get this to testing

**Pull request closed without merging**